### ISSUE TITLE
Trakt add-on adds multiple duplicate plays

### ISSUE THREAD

**Issue by Waldner11** [razzeee/script.trakt#468]
Hi,
I think this is an issue with the trakt add-on so I'm putting it here.
Trakt add-on version 3.2.0 / Kodi version 17.6.0

So basically my issue is that I got a ton of plays that gets added to my library, often it's duplicate plays of shows that I've already watched (at the same time) other times it's shows that I've collected that gets watched in trakt sometime after (in the next day or so). While I've never rewatched them. I'm talking about like 40 days of plays that gets added, only for tv shows. 
Now this sort of issue already happened to me in the past, 4 or 5 times at the minimum. But these couple of days I think it's in some kind of loop and it's happening like 4 times in a row. Each time I launch Kodi it seems there's a brand new load of plays that gets added.

I've never had the "Mark tv episodes from trakt history as watched in Kodi" option ticked, so the add-on should never send more than a couple plays a minute. Just when I finish watching an item.

When looking at the kodi debug log, I've seen only these "trakt related" but multiple times :
00:01:57.989 T:6168 WARNING: [script.trakt] trakt.interfaces.base: unable to parse JSON response: Expecting value: line 1 column 606763 (char 606762)
00:02:00.571 T:6168 WARNING: [script.trakt] trakt.interfaces.base: Request failed: "POST /sync/history" - 500: "Server error" (Server Error)

I've already sent a request to the trakt support here (https://trakt.tv/forums/media-centers/topics/tons-of-plays-added-at-the-same-time-most-likely-by-kodi?pagination_param=1#post-12171)
My username is Waldner11, I'm not sure how close you are to the trakt support.

I'm willing to help, debug, try a debug/custom version of the add-on, provide any information necessary to help fixing this bug. 

Thanks for your help, as I've said if you need more information I'm willing to assist.

**Comment by razzeee** [comment]
Let's wait for justin to get back.

**Comment by Waldner11** [comment]
I've talked to him yesterday, did he inform you on what we were talking about?

**Comment by razzeee** [comment]
No, we don't talk on a regular basis. As I'm just a volunteer and not affiliated with trakt at all.

**Comment by Waldner11** [comment]
Maybe I could forward you the mail.

Télécharger Outlook pour Android<https://aka.ms/ghei36>

________________________________

**Comment by deadlyjinx** [comment]
I can confirm this bug, also with Trakt add-on version 3.2.0 / Kodi version 17.6.0.

It seems that this only happens with movies/shows on my library and I'm using https://huere.net/trakt/duplicate_removal/ to remove the duplicates.

**Comment by dm2912** [comment]
can confirm same here. its **only with library items** and it happens whenever you pause and resume a show/movie

so if i pause three times during a show it submits 3 extra scrobbles



alternate option for removing duplicates is this python script that works really well and quickly
https://github.com/anthofo/trakt-tv-duplicates-removal

**Comment by razzeee** [comment]
Can't reproduce this with 18.4
